website/docs/install.md

```diff
@@ -97,26 +97,13 @@ Add the following:
 
 ### [VSCode Neovim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim) users
 
-Add the following to your `init.vim`:
```

Comment:
Is there any reason to still keep these instructions? Maybe some users want to keep configuration in their Neovim configuration files?

Reply:
Yes, exactly.
I would like to keep both the approaches documented. Can you edit the PR to put your approach before the existing one?
You can describe the existing approach as "alternative".